### DISCUSSION TITLE
Extended Resolution - digit direction

### DISCUSSION THREAD

**Discussion by bucklevision** [jomjol/AI-on-the-edge-device#1863] [Q&A]
Hi Folks, what direction does the extended resolution assume digits will move? I think it might be opposite from my own meter. The meter numbers increase by the digits moving downwards. Right now I'm just beyond 3 on the way to 4, and it's reading 2.9. 

I think this is complicated by having negative rates enabled. 

Can anyone clarify if the algo can accommodate both of these settings together? I have a very low meter movement (1 sample per 15 mintues, maybe one digit move every 12 hours, but potentially in both directions).

**Comment by caco3** [comment]
Yes, I think only meters with digits rotating from bottom to top work.
Maybe you could try the different models.

On the other side, the analogue pointers can be configured for both directions.

An animation of a "normal" meter can be found here: https://jomjol.github.io/AI-on-the-edge-device-docs/Demo-Mode/#animation
